SPECS/kernel/kernel.spec

```diff
@@ -15,7 +15,6 @@ Source2:        config_aarch64
 # Arm64 HyperV support required patch
 Patch0:         ver5_4_72_arm64_hyperv_support.patch
 Patch1:         efi-libstub-tpm-enable-tpm-eventlog-function-for-ARM.patch
-Patch2:         fix-gui-installer-mmap.patch
```

Comment:
Have you also removed the file itself? I cannot find it in this PR.

Reply:
Good catch. The patch file is still present in the directory. Will remove.